Changes

```diff
@@ -118,6 +118,10 @@ _______________
 - #12959, #13055: Avoid an internal error on recursive module type inconsistency
   (Florian Angeletti, review by Jacques Garrigue and Gabriel Scherer)
 
+- #13001: do not read_back entire shapes to get aliases' uids when building the
+  usages index
+  (Ulysse Gérard, review by Gabriel Scherer)
```

Comment:
Let's also add Nathanaëlle.

Reply:
And let's move the entry in the 5.2 section too.